zeppelin-distribution/src/bin_license/licenses/LICENSE-nvd3.js-1.1.15-beta

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2011, 2012 Novus Partners, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
```

Comment:
This file can be removed as this is Apache 2 licence and there is no need to put it there one more time.